internal/utils/http/http.go

```diff
@@ -160,7 +151,20 @@ func SendEmptyResponse(w http.ResponseWriter, r *http.Request) {
 }
 
 //SendErrorResponse Sends error response as required by Firebase SDK.
-func SendErrorResponse(w http.ResponseWriter, r *http.Request, error rpccode.Code, message string) {
+func SendErrorResponse(w http.ResponseWriter, r *http.Request, err error) {
+	logger := logging.FromContext(r.Context())
+
+	var mr, ok = err.(errors.ErouskaError)
+
+	if ok {
+		sendErrorResponse(w, r, mr.Code(), mr.Error())
+	} else {
+		logger.Warnf("Cannot handle request due to unknown error: %+v", err.Error())
+		sendErrorResponse(w, r, rpccode.Code_INTERNAL, err.Error())
+	}
```

Comment:
can this be if { .. return} without `else` pls? :D

Reply:
As you wish :man_shrugging: